README.md

```diff
@@ -15,15 +15,15 @@
 
 ## Documentation
 
-You can find installation instructions and detailed instructions on how to use this package at the [dedicated documentation site](https://docs.ark.io/sdk/cryptography/go.html).
+You can find installation instructions and detailed instructions on how to use this package at the [documentation site](https://docs.ark.io/sdk/cryptography/usage.html).
 
 ## Security
 
 If you discover a security vulnerability within this package, please send an e-mail to security@ark.io. All security vulnerabilities will be promptly addressed.
 
 ## Credits
 
-This project exists thanks to all the people who [contribute](../../contributors).
+This project exists thanks to all the people who [contribute](./contributors).
```

Comment:
This is intentionally `../..` as it goes through some internal routing of GitHub that eventually ends up at https://github.com/ArkEcosystem/go-crypto/graphs/contributors.

Reply:
Maybe that was the case in the past, but now there is no "internal" routing (whatever that could mean). It is a basic HTTP redirect:

```
$ curl --head https://github.com/ArkEcosystem/go-crypto/contributors
HTTP/1.1 302 Found
Status: 302 Found
Location: https://github.com/ArkEcosystem/go-crypto/graphs/contributors
...
```

We could also point the link directly to `./graphs/contributors`.

Reply:
Changing to that is also fine.

Reply:
Done.